src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.targets

```diff
@@ -40,22 +40,22 @@
   <ItemGroup Condition=" $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '@MAUI_PREVIOUS_DOTNET_VERSION@')) and ('$(SkipMauiWorkloadManifest)' != 'true') ">
     <KnownFrameworkReference
         Update="Microsoft.Maui.Core"
-        DefaultRuntimeFrameworkVersion="@MAUI_PREVIOUS_DOTNET_RELEASED_NUGET_VERSION@"
-        LatestRuntimeFrameworkVersion="@MAUI_PREVIOUS_DOTNET_RELEASED_NUGET_VERSION@"
-        TargetingPackVersion="@MAUI_PREVIOUS_DOTNET_RELEASED_NUGET_VERSION@"
+        DefaultRuntimeFrameworkVersion="$(MauiVersion)"
+        LatestRuntimeFrameworkVersion="$(MauiVersion)"
+        TargetingPackVersion="$(MauiVersion)"
     />
```

Comment:
Is there a test we could add that would show this "fixed" something? Does the `net8.0` branch build a `net7.0` MAUI app to make sure it builds & runs?

Reply:
It does. All templates are build with "default", netCurrent and netPrevious TFMs

Reply:
I guess we don't have any tests that would set `$(MauiVersion)`? The property has only been tested manually so far?

Reply:
I'll add some tests.